### PR TITLE
Fix an issue that prevents startup if filters is not set

### PR DESCRIPTION
**Pull request by cagataygurturk**

### CODE REVIEW

**Comment by etiennetremel** [comment]
@andrei-pavel would you mind reviewing double checking this?

File: main.go [hunk 1]
```diff
@@ -101,17 +101,19 @@ func main() {
...
-	*filters = strings.Replace(*filters, " ", "", -1)
-	pairs := strings.Split(*filters, ";")
-	for _, pair := range pairs {
-		keyValue := strings.Split(pair, ":")
-
-		// Check format.
-		if len(keyValue) != 2 {
-			panic(fmt.Sprintf("filter '%v' should be of the form `label_key: label_value`", keyValue))
-		}
+	if filters != "" {
```
Comment (andrei-pavel) [inline main.go]: `filters` is a pointer to a string provided by `kingpin`, it needs to be dereferenced
```
	if *filters != "" {
```

**Comment by andrei-pavel** [comment]
Code doesn't compile at the moment. Otherwise, the fix is spot on. If suggested change gets implemented, you're good to merge.

Until then, workaround for no filter is using `-f ':'`.

My apologies for this careless regression. Looked more closely for other issues. There seem to be none.

**Comment by cagataygurturk** [comment]
Now it should be fixed

**Pull request merged**